Foundation/NSData.swift

```diff
@@ -217,7 +217,11 @@ open class NSData : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
     }
     
     open var bytes: UnsafeRawPointer {
-        return UnsafeRawPointer(CFDataGetBytePtr(_cfObject))
+        guard let bytes = CFDataGetBytePtr(_cfObject) else {
+            //This could occure on empty data being encoded.
+            return UnsafeRawPointer([])
+        }
+        return UnsafeRawPointer(bytes)
```

Comment:
I think the return value here might not be safe since the array address is invalid after the return. In reality the "proper fix" is for us to change the signature of bytes